packages/react-core/package.json

```diff
@@ -40,23 +40,23 @@
     "react": ">= 16.14.0"
   },
   "devDependencies": {
+    "@aws-amplify/eslint-config-amplify-ui": "0.0.0",
```

Comment:
Can we scope this to just the `"@aws-amplify/eslint-config-amplify-ui"` packages? The other sorting feels kind of arbitrary

Reply:
technically yes, but I don't want to make another PR only to sort it

Reply:
What i mean is that the ordering changes for packages not relevant to this PR feel random, and imo are no longer sorted:
```
// previously @testing-library/react is ordered after @testing-library/react-hooks
"@testing-library/react": "^12.0.0",
"@testing-library/react-hooks": "^7.0.2",

// where as in the PR, now @testing-library/react-hooks is ordered after @testing-library/react
"@testing-library/react-hooks": "^7.0.2",
"@testing-library/react": "^12.0.0",
```

Reply:
I see...It was sorted by a VS Code extention, I'm not sure what algorithm it uses (the charCode of `"` is 34, `-` is 45 though), but I think as long as it consistent, it should not be a big deal?

Reply:
Oh, I see what you mean. I thought those were not sorted before. They were sorted in a different way.